docs/benefits.md

```diff
@@ -332,6 +332,19 @@ Yes! Benepass is able to track and manage ‘payment’ systems for large purcha
 
 Check out a more robust list [here](https://docs.google.com/spreadsheets/d/1EAmO6874G_FqyVAGJTXG-eU3TNIBuNo8WiEHnwS79vY/edit#gid=0). This isn’t an exhaustive list but we will build it out over time as questions arise.
 
+## Wellness Program
```

Comment:
Are you trying to move the wellness program within the stipend section, above the professional dev section? Because of how the headers are used, this is going to throw off the structure of the page.  See how it's currently structured. Lmk what the reason is to move the Wellness Program within Stipends and we can see if a different header structure is needed.

<img width="245" alt="E11CBAD6-6E30-448C-BB61-1A6303BBF6DF" src="https://github.com/dbt-labs/handbook/assets/127452841/f1850071-c2a4-419d-a5c7-490f3bcbbabc">